content/tutorial-regular-keys.md

```diff
@@ -0,0 +1,764 @@
+* [Removing or changing a regular key pair](#removing-or-changing-a-regular-key-pair)
+
+
+## Assigning a Regular Key Pair
+
+This tutorial walks you through the steps required to assign a regular key pair to your account:
+
+1. [Generate a key pair](#1-generate-a-key-pair)
+2. [Assign the key pair to your account as a regular key pair](#2-assign-the-key-pair-to-your-account-as-a-regular-key-pair)
+3. [Verify the regular key pair](#3-verify-the-regular-key-pair)
+4. [Explore next steps](#4-explore-next-steps)
+
+
+### 1. Generate a Key Pair
+
+Use the [`wallet_propose`](reference-rippled.html#wallet-propose) method to generate the key pair that you'll assign to your account as a regular key pair.
+
+#### Request Format
+
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+  "method": "wallet_propose"
+}
+```
+
+*JSON-RPC*
+
+```
+{
+  "method": "wallet_propose"
+}
+```
+
+*Commandline*
+
+```
+#Syntax: wallet_propose
+rippled wallet_propose
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+#### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+  "result": {
+    "account_id": "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+    "key_type": "secp256k1",
+    "master_key": "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+    "master_seed": "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+    "master_seed_hex": "966C0F68643EFBA50D58D191D4CA8AA7",
+    "public_key": "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+    "public_key_hex": "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E"
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+*JSON-RPC*
+
+```
+{
+    "result": {
+        "account_id": "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+        "key_type": "secp256k1",
+        "master_key": "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+        "master_seed": "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+        "master_seed_hex": "966C0F68643EFBA50D58D191D4CA8AA7",
+        "public_key": "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+        "public_key_hex": "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E"
+        "status": "success"
+    }
+}
+```
+
+*Commandline*
+
+```
+{
+   "result" : {
+      "account_id" : "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+      "key_type" : "secp256k1",
+      "master_key" : "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+      "master_seed" : "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+      "master_seed_hex" : "966C0F68643EFBA50D58D191D4CA8AA7",
+      "public_key" : "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+      "public_key_hex" : "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E",
+      "status" : "success"
+   }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+In the next step, you'll use the `account_id` from this response to assign the key pair as a regular key pair to your account.
```

Comment:
Tell them to save the `master_seed` value somewhere securely also. (Everything else they can forget about.)